ivy/functional/frontends/paddle/math.py

```diff
@@ -458,6 +458,36 @@ def rsqrt(x, name=None):
     return 1 / ivy.sqrt(x)
 
 
+@with_supported_dtypes(
```

Comment:
Replace this tag with `@with_unsupported_dtypes({"2.5.1 and below": ("float16", "bfloat16")}, "paddle")`